airflow/configuration.py

```diff
@@ -688,18 +680,13 @@ def as_dict(self, display_source=False, display_sensitive=False):
 
         return cfg
 
-    def load_test_config(self):
-        """
-        Load the unit test configuration.
-
-        Note: this is not reversible.
-        """
-        # override any custom settings with defaults
-        self.read_string(parameterized_config(DEFAULT_CONFIG))
-        # then read test config
-        self.read_string(parameterized_config(TEST_CONFIG))
-        # then read any "custom" test settings
-        self.read(TEST_CONFIG_FILE)
+    def load_or_save_config_file(self, config_file, config_string=None):
```

Comment:
This should be split into two methods, `load_config_file` and `save_config_file`. Otherwise it will be extremely confusing to figure out what it's doing when reading code.